hipcheck/Cargo.toml

```diff
@@ -86,6 +86,9 @@ http = "1.1.0"
 dialoguer = "0.11.0"
 tabled = "0.15.0"
 fs_extra = "1.3.0"
+tonic = "0.12.1"
+prost = "0.13.1"
+tonic-build = { version = "0.12.1", features = ["prost"] }
```

Comment:
Does `tonic` not re-export `prost` or provide a feature for building? Seems surprising to need to import all 3.

Reply:
Oh also is `tonic-build` a requirement at runtime, or just build time? I see it's also a build dependency.

Reply:
removing `prost` from the Cargo.toml causes compilation to fail. I don't see anything in the list of disabled `tonic` features that indicates such a feature, they're all compression- or TLS-related.

Reply:
good catch on the `tonic-build` in `dependencies`, will remove

Reply:
Interesting! Looks like you're right, I'm not seeing anything that re-exports `prost`, so I think just removing `tonic-build` is all we can do.